include/calibrationLib.py

```diff
@@ -251,4 +258,48 @@ def query_betas(connection, studyId, filterZero=True, filename="data/calibration
         csvfile.write(header)
         for row in rows:
             line = ','.join(str(row[ndx]) for ndx in range(0, len(row)))
-            csvfile.write("{}\n".format(line))
\ No newline at end of file
+            csvfile.write("{}\n".format(line))
+
+def goodness_of_variance_fit(array, classes):
+    # get the break points
+    classes = jenkspy.jenks_breaks(array, classes)
+
+    # do the actual classification
+    classified = np.array([classify(i, classes) for i in array])
+
+    # max value of zones
+    maxz = max(classified)
+
+    # nested list of zone indices
+    zone_indices = [[idx for idx, val in enumerate(classified) if zone + 1 == val] for zone in range(maxz)]
+
+    # sum of squared deviations from array mean
+    sdam = np.sum((array - array.mean()) ** 2)
+
+    # sorted polygon stats
+    array_sort = [np.array([array[index] for index in zone]) for zone in zone_indices]
+
+    # sum of squared deviations of class means
+    sdcm = sum([np.sum((classified - classified.mean()) ** 2) for classified in array_sort])
+
+    # goodness of variance fit
+    gvf = (sdam - sdcm) / sdam
+
+    return gvf, classes
+
+def classify(value, breaks):
+    for i in range(1, len(breaks)):
+        if value < breaks[i]:
+            return i
+    return len(breaks) - 1
+
+# getting data ready for binning
+def bin_asc(filename):
```

Comment:
This is a poor function name since it implies that the result of the function will be a binned data set, not a data set that is ready for binning.